client/js/render.js

```diff
@@ -72,6 +75,12 @@ function buildChatMessage(data) {
 	const msg = $(templates[template](data.msg));
 	const text = msg.find(".text");
 
+	if (data.msg.previews.length) {
+		data.msg.previews.forEach((preview) => {
```

Comment:
We can't because of this rule: http://eslint.org/docs/rules/brace-style.
Alternative is to do `data.msg.previews.forEach((preview) => renderPreview(preview, msg));`, which has an implicit return value that we shouldn't convey, so this is actually the best option IMO.